docs: add documentation stating that we do not support Web in React Native SDK (#1776)
GetStream/stream-chat-react-native: docusaurus/docs/reactnative/basics/getting_started.mdx

| 0.64         | 3.6.2                                   | 2.2.0                                  |
| 0.63         | 3.0.0                                   | 2.2.0                                  |

#### Platform compatibilities

We only support **Android** and **iOS** as platforms for the React Native SDK. We do not support **Web** currently, but this is in our backlog, and we plan to take this up in the future.

> Meanwhile, you can use our [React SDK](https://getstream.io/chat/docs/sdk/react/) for Web. They almost follow the same API and Component convention as the React Native SDK, so you should be able to use them efficiently.

Reviewer: ```suggestion
As an alternative, you can use Stream's [React SDK](https://getstream.io/chat/docs/sdk/react/) for Web. The React SDK follows the same API and Component convention and is very similar to the React Native SDK.
```
